lib/commands/new.js

```diff
@@ -85,8 +85,32 @@ module.exports = async (cwd, dir, ctx) => {
         'directories'
     ];
 
+    const isDep = (name) => finalPkg.dependencies.hasOwnProperty(name);
+    const isDevDep = (name) => finalPkg.devDependencies.hasOwnProperty(name);
     const cmp = (x, y) => {
 
+        // Full-coverage here depends on input package file having keys to trigger
+        // all possible paths. The boilerplate's doesn't
+        // Only cases tested by package.json's entire set of keys is all false
+        // and half-true both ways (both deps or both dev deps).
+        /* $lab:coverage:off$ */
+        if (
+            (isDep(x.key) && isDep(y.key)) ||
+            (isDevDep(x.key) && isDevDep(y.key))
+        ) {
+            // boilerplate package.json deps are already alphabetical,
+            // so, the if clause is always true, meaning its only half-covered b/c false branch not tripped
+            // (this registers as missing coverage in node 12, but not 10)
+            // and the else branch is uncovered (never reached)
+            if (x.key > y.key) {
+                return 1;
+            }
+            else if (x.key < y.key) {
```

Comment:
I think there's a dead path here (`x.key < y.key` always `true`), covered-up by the turned-off coverage:
```suggestion
            else {
```

Could use a double-check on that, though: does that seem right?

Reply:
Gah, sorry, yea, that sounds right. This was me being confused on how to deal with the `x.key === y.key` case. I'll clean this up!